### ISSUE TITLE
[Question] Can't make api-platform work with Symfony 4

### ISSUE THREAD

**Issue by sstok** [api-platform/core#1452]
I'm trying to fix the RollerworksSearch Symfony 4 compatibility, but for some reason I can't get the API-Platform to work with Symfony 4.

https://travis-ci.org/rollerworks/RollerworksSearchBundle/jobs/291052328

```
 Caused by
 ApiPlatform\Core\Exception\RuntimeException: There is no builtin action for the collection GET operation. You need to define the controller yourself.
 
 /Users/sebastiaanstok/Sites/rollerworks/search-bundle/vendor/api-platform/core/src/Bridge/Symfony/Routing/ApiLoader.php:185
 /Users/sebastiaanstok/Sites/rollerworks/search-bundle/vendor/api-platform/core/src/Bridge/Symfony/Routing/ApiLoader.php:90
```

I have no idea why it's not working. I'm doing something wrong or is Symfony 4 support still experimental?

**Comment by GuilhemN** [comment]
see https://github.com/api-platform/core/pull/1453

**Comment by meyerbaptiste** [comment]
Fixed since `api-platform/core` `2.1.2`.
